portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/FormEvents.testcase

```diff
@@ -840,7 +840,7 @@ definition {
 		}
 
 		task ("Wait for sending requests") {
-			Pause(locator1 = "5000");
+			ACUtils.waitForSendingRequest();
 		}
```

Comment:
Hey @calvinctvo, I think this pause won't be necessary because we'll interact with the DXP in the next step.